tests/e2e/specs/proposal.spec.js

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable ui-testing/no-disabled-tests */
+describe('Make Proposal Tests', () => {
+  let startTime;
+  context('PSM tests', () => {
+    it('should setup two econ committee member wallets', () => {
+      cy.setupWallet({
+        secretWords:
+          'purse park grow equip size away dismiss used evolve live blouse scorpion enjoy crunch combine day second news off crowd broken crop zoo subject',
+        walletName: 'gov1',
+      });
+      cy.setupWallet({
+        secretWords:
+          'tilt add stairs mandate extra wash choose fashion earth feature reopen until move lazy carbon pledge sure own comfort this nasty clap tower table',
+        walletName: 'gov2',
+      });
+    });
+
+    it('should connect to wallet', () => {
+      cy.visit('/?agoricNet=local');
+      cy.acceptAccess();
+    });
+
+    it('should have value of 1000 on psm inter', () => {
+      cy.origin('https://psm.inter.trade', () => {
```

Comment:
Not sure if it's good to depend on the live UI here since a service disruption could impede workflows. Although, even when using a local build of PSM and connecting to a local chain, the app will make a request to https://wallet.agoric.app/network-config. Maybe it's enough to just check that the history item shows up with the new value, below the check for 'Change Accepted', instead of looking at the PSM app.

cc @turadg

Reply:
Good point, @samsiegart .  Could the e2e tests intercept https://wallet.agoric.app/network-config to provide a local result? 

I don't see this as a blocking concern for having these tests in CI, but it should be addressed before required green to merge. (I.e. adding it to branch protection)

Reply:
sounds good. I'll remove the check from the external PSM URL